.gitignore

```diff
@@ -1,5 +0,0 @@
-.idea/
```

Comment:
You've pushed the .idea and __pycache__ folders to your repo, which should be excluded.